src/platform/packages/shared/kbn-unified-metrics-grid/src/components/toolbar/dimensions_selector.tsx

```diff
@@ -131,6 +178,10 @@ export const DimensionsSelector = ({
       options={options}
       singleSelection={false}
       onChange={handleChange}
+      popoverTitle={i18n.translate('metricsExperience.dimensionsSelector.label', {
```

Comment:
There’s no title in the designs. Do we need one?

Reply:
True, I removed it. It was also adding the `aria-label`s using the title content, and unfortunately, we won't get them now 😞